### PR TITLE
Build Tools: Maven

### PR DESCRIPTION
**Pull request by denisdimitrov**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    </modules>
+
+    <dependencies>
+        <!-- Logs lib-->
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+        </dependency>
+        <!-- Testing lib-->
```
Comment (EmilGeorgiev) [inline pom.xml]: add row before comment.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+            <version>1.2.17</version>
+        </dependency>
+    </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <!--Building JAR-->
+                <groupId>org.apache.maven.plugins</groupId>
```
Comment (EmilGeorgiev) [inline pom.xml]: i cant see ```jarjar``` plugin

**Comment by denisdimitrov** [comment]
Added jarjar.

**Comment by EmilGeorgiev** [comment]
LGTM

**Pull request merged**